ThirdPartyAdapters/vungle/vungle/build.gradle

```diff
@@ -31,10 +31,15 @@ android {
 }
 
 dependencies {
-    implementation ('com.vungle:publisher-sdk-android:6.10.2') {
+
+    implementation ('com.github.vungle:vungle-android-sdk:6.10.3-RC3') {
         transitive=true
     }
 
+//    implementation ('com.vungle:publisher-sdk-android:6.10.3') {
+//        transitive=true
+//    }
+
     implementation 'androidx.annotation:annotation:1.2.0'
     implementation 'com.google.android.gms:play-services-ads:20.3.0'
```

Comment:
we should bump to use latest admob sdk dependency
https://developers.google.com/admob/android/sdk